xtra-libp2p-ping/src/pong.rs

```diff
@@ -0,0 +1,30 @@
+use crate::protocol;
+use async_trait::async_trait;
+use tokio_tasks::Tasks;
+use xtra_libp2p::NewInboundSubstream;
+use xtra_productivity::xtra_productivity;
+
+#[derive(Default)]
+pub struct Actor {
+    tasks: Tasks,
+}
+
+#[xtra_productivity(message_impl = false)]
+impl Actor {
+    async fn handle(&mut self, message: NewInboundSubstream) {
```

Comment:
❓Why does pong not return a `pong`? 

Isn't the point of a ping protocol that you get a pong from the other party when sending a ping? Am I missing something?